tests/test_diff.cpp

```diff
@@ -559,80 +559,80 @@ TEST_CASE("shape-neumann-nodes", "[test_adjoint]")
 // 	}
 // }
 
-TEST_CASE("shape-pressure-neumann-nodes", "[test_adjoint]")
-{
-	const std::string path = POLYFEM_DATA_DIR + std::string("/differentiable/input/");
-	json in_args;
-	load_json(path + "shape-pressure-neumann-nodes.json", in_args);
-	auto state_ptr = create_state_and_solve(in_args);
-	State &state = *state_ptr;
+// TEST_CASE("shape-pressure-neumann-nodes", "[test_adjoint]")
```

Comment:
a lot of comments?

Reply:
Yea, so I'm waiting on this to get merged so I can send another PR with some optimization objectives that will increase the code coverage.